textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToGramQueryTranslatorTest.java

```diff
@@ -17,133 +17,321 @@ private void printTranslatorResult(String regex) {
 		
 		System.out.println("regex: "+regex);
 		System.out.println("boolean expression: "+exactQuery.getLuceneQueryString());
+		
 		System.out.println("query tree: ");
```

Comment:
Need to board now (in LAX) :-)

Thank you Jianfeng for your help on this PR.

On Wed, Jun 8, 2016 at 11:43 PM, Jianfeng Jia <notifications@github.com>
wrote:

> In
> textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToGramQueryTranslatorTest.java
> <https://github.com/TextDB/textdb/pull/127#discussion_r66390270>:
>
> > @@ -17,133 +17,321 @@ private void printTranslatorResult(String regex) {
> >  		
> >  		System.out.println("regex: "+regex);
> >  		System.out.println("boolean expression: "+exactQuery.getLuceneQueryString());
> > +		
> >  		System.out.println("query tree: ");
>
> Review in the air? 👍
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/TextDB/textdb/pull/127/files/8416c34354ddc5888e29ad41e6b6ac3c7b2e7f0d#r66390270>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMU9nU8U2HUxaT3B41Gm2BgK9kTNFIwLks5qJ7YigaJpZM4IxnIT>
> .
>